saleor/checkout/views/__init__.py

```diff
@@ -55,3 +56,12 @@ def summary_view(request, checkout):
         return summary_without_shipping(request, checkout)
     else:
         return anonymous_summary_without_shipping(request, checkout)
+
+
+@validate_cart
+def login(request, checkout):
```

Comment:
What's the idea behind having a second login page? I would expect at least a docstring with a good explanation of why and how it works.